Detectors/GlobalTrackingWorkflow/src/tof-matcher-workflow.cxx

```diff
@@ -125,7 +125,7 @@ WorkflowSpec defineDataProcessing(ConfigContext const& configcontext)
   LOG(debug) << "Store all matchables = " << writeMatchable;
 
   //GID::mask_t alowedSources = GID::getSourcesMask("TPC,ITS-TPC");
-  GID::mask_t alowedSources = GID::getSourcesMask("TPC,ITS-TPC,TPC-TRD,ITS-TPC-TRD");
+  GID::mask_t alowedSources = GID::getSourcesMask("TPC,ITS-TPC,TPC-TRD,ITS-TPC-TRD,ITS");
```

Comment:
with https://github.com/AliceO2Group/AliceO2/blob/b1a6d3674f4a1d8d526710296552f0bfd75550db/Detectors/GlobalTrackingWorkflow/src/tof-matcher-workflow.cxx#L59C6-L59C156 have "all" by default, the ITS-TOF matching will be enabled by default. Could you change this default `track-sources` to explicit list w/o ITS?

Reply:
done